tensorflow_model_optimization/python/core/sparsity/keras/pruning_utils.py

```diff
@@ -159,11 +159,48 @@ def factorized_pool(input_tensor,
 
   return tf.squeeze(tf.transpose(width_pooling, perm=[0, 1, 3, 2]))
 
+
+def normalise_tuple(value, name):
```

Comment:
I am not sure whether this function need to be a shared utility. Ideally, we can  force the input type using typing arguments. 

If you want to use this, please name the function more explicit. From this name, we cannot know it will return a tuple of 2 ints. just suggestion - maybe_convert_to_tuple_of_two_ints

Reply:
Done